### PR TITLE
Brand switcher

### PR DESCRIPTION
**Pull request by maryam-mv**

### CODE REVIEW

File: website/src/components/brand-picker/brand-picker.js [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
-		<div css={{ padding: '100px' }}>
-			<h1 css={{ textAlign: 'center' }}>Select your default brand</h1>
+		<div
+			css={{
+				padding: '100px',
+				fontFamily:
+					'-apple-system, BlinkMacSystemFont, "Helvetica Neue", Helvetica, Arial, sans-serif',
+			}}
+		>
+			<p as="h1" css={{ textAlign: 'center', fontSize: '20px', color: 'rgb(45, 55, 62)' }}>
```
Comment (MadeByMike) [inline website/src/components/brand-picker/brand-picker.js]: Why p as h1?

File: website/src/components/brand-picker/brand-picker.js [hunk 2]
```diff
@@ -2,13 +2,22 @@
...
 import { useBrandSwitcher } from '../providers/brand-switcher';
+
 export const BrandPicker = () => {
 	const { brands } = useBrandSwitcher();
 	const [hovered, setHovered] = useState(false);
 
 	return (
-		<div css={{ padding: '100px' }}>
-			<h1 css={{ textAlign: 'center' }}>Select your default brand</h1>
+		<div
```
Comment (MadeByMike) [inline website/src/components/brand-picker/brand-picker.js]: Why is the font not inherited from a parent?

File: website/src/components/brand-switcher/brand-switcher.js [hunk 3]
```diff
@@ -1,60 +1,194 @@
...
 				<Link href={`/?b=${brandName}`}>
 					<a>
 						<Logo />
 					</a>
 				</Link>
 			</div>
-			<label
+
+			<div
+				tabindex="0"
```
Comment (MadeByMike) [inline website/src/components/brand-switcher/brand-switcher.js]: Disable tab?

Comment (MadeByMike) [reply website/src/components/brand-switcher/brand-switcher.js]: Looks like tabbing works but selecting an option with ENTER doesn't

File: website/src/components/brand-picker/brand-picker.js [hunk 4]
```diff
@@ -1,19 +1,31 @@
...
+			}}
+		>
+			<h1>Select your default brand</h1>
+			<p css={{ fontSize: '20px', color: 'rgb(45, 55, 62)' }}>
+				Once you've made this selection you can change it at any time.
+			</p>
 			<ul
 				css={{
 					margin: '0 auto',
 					padding: '40px 20px',
```
Comment (dominikwilkowski) [inline website/src/components/brand-picker/brand-picker.js]: Should use `rem`
```suggestion
					padding: '2.5rem 1.25rem',
```

File: website/src/components/brand-picker/brand-picker.js [hunk 5]
```diff
@@ -1,19 +1,31 @@
...
-					maxWidth: '1160px',
+					maxWidth: '1080px',
```
Comment (dominikwilkowski) [inline website/src/components/brand-picker/brand-picker.js]:
```suggestion
					maxWidth: '67.5rem',
```

File: website/src/components/brand-picker/brand-picker.js [hunk 6]
```diff
@@ -26,10 +38,14 @@ export const BrandPicker = () => {
 						<li
 							key={i}
 							css={{
-								flexBasis: '250px',
+								flexBasis: '100%',
+								'@media min-width(600px)': {
+									flexBasis: '100%',
+									maxWidth: 250,
```
Comment (dominikwilkowski) [inline website/src/components/brand-picker/brand-picker.js]: Let's make units explicit:
```suggestion
									maxWidth: '15.625rem',
```

File: website/src/components/brand-picker/brand-picker.js [hunk 7]
```diff
@@ -26,10 +38,14 @@ export const BrandPicker = () => {
...
+								},
+								margin: 10,
```
Comment (dominikwilkowski) [inline website/src/components/brand-picker/brand-picker.js]:
```suggestion
								margin: '0.625rem',
```

**Pull request merged**